README.md

```diff
@@ -11,45 +11,79 @@ The project can use **SDL** but it can be easily relaced by any other built-in L
 ## Get started
 
 ### Get the PC project
+
 Clone the PC project and the related sub modules:
 
 ```bash
 git clone --recursive https://github.com/lvgl/lv_port_pc_vscode
 ```
 
 ### Install SDL and build tools
+
 You can download SDL from https://www.libsdl.org/
 
-#### Linux 
-Copy below in the Terminal:  
+#### Linux
+
+Copy below in the Terminal:
 For Ubuntu
-```bash 
+
+```bash
 sudo apt-get update && sudo apt-get install -y build-essential libsdl2-dev cmake
 ```
-For ArchLinux  
-```bash 
+
+For ArchLinux
+
+```bash
 sudo pacman -Syu && sudo pacman -S sdl2 libsdl2-devel sdl2_mixer sdl2-devel base-devel gcc make
 ```
 
 ## Usage
 
 ### Visual Studio Code
+
 1. Be sure you have installed [SDL and the build tools](#install-sdl-and-build-tools)
-1. Open the project by double clicking on `simulator.code-workspace` or opening it with `File/Open Workspace from File`
-2. Install the recommended plugins
-3. Click the Run and Debug page on the left, and select `Debug LVGL demo with gdb` from the drop-down on the top. Like this:
+2. Open the project by double clicking on `simulator.code-workspace` or opening it with `File/Open Workspace from File`
+3. Install the recommended plugins
+4. Click the Run and Debug page on the left, and select `Debug LVGL demo with gdb` from the drop-down on the top. Like this:
 ![image](https://github.com/lvgl/lv_port_pc_vscode/assets/7599318/f527b235-5718-4949-b5f0-bd807b3a64ba)
-4. Click the Play button or hit F5 to start debugging.
-   
-**ArchLinux User**  
-VSCode does not officially provide an installation package under Arch, you need to use the AUR manager `paru` to install it.  
+5. Click the Play button or hit F5 to start debugging.
+
+#### ArchLinux User
+
+VSCode does not officially provide an installation package under Arch, you need to use the AUR manager `paru` to install it.
 The command is as follows:
+
 ```bash
 paru -S visual-studio-code-bin
 ```
 
+#### macOS
+
+Apple's default clang does not support the `-fsanitize=leak` flag. You might see this error when building:
+
+```clang
+clang: error: unsupported option '-fsanitize=leak' for target 'arm64-apple-darwin23.5.0'
+make[2]: *** [CMakeFiles/main.dir/main/src/main.c.o] Error 1
+make[1]: *** [CMakeFiles/main.dir/all] Error 2
+make: *** [all] Error 2
+```
```

Comment:
With https://github.com/lvgl/lv_port_pc_vscode/commit/e08bfca8aab3fe548075ae2176751fa6186de812 this error shouldn't be shown.